apps/rush-lib/src/logic/PackageChangeAnalyzer.ts

```diff
@@ -129,10 +129,20 @@ export class PackageChangeAnalyzer {
     //  });
     // }
 
+    // Determine the current variant from the link JSON.
```

Comment:
>current [](start = 21, length = 7)

We should have a public API for reading **current-variant.json**, for tools that need to know what variant name they're supposed to be working with.

Reply:
Added a `currentInstalledVariant` property to `RushConfiguration`.